docs/b2c-digital-goods.md

```diff
@@ -0,0 +1,9 @@
+## Digital Goods (B2C)
+
+<img src="images/digital-goods-b2c.png" alt="Digital Goods (B2C)">
+
+This part of the plugin handles taxes for Digital Goods for Consumers. Currently, the plugin offers 2 options for this class of taxes.
```

Comment:
"for Digital Goods for Consumers when selling to the EU" and add the link that's in the plugin